development/image-syncer/external-images.yaml

```
@@ -41,10 +41,7 @@ images:
 - source: "quay.io/prometheus/alertmanager:v0.23.0"
 - source: "quay.io/prometheus/prometheus:v2.32.1"
 - source: "quay.io/service-manager/sb-proxy-k8s:v0.9.1"
-- source: "nats@sha256:ae7411548aaccfbe8c77b0f055c26ca31e0f446b32d5e0ac2057d071c679adab"
-  tag: "2.7.4-alpine"
-- source: "nats@sha256:bed4f10253c279b665399962dd8a61466115c3e1e9acd74470e29f955c4dc923"
-  tag: "2.7.4-alpine-2022-04-05"
+- source: "nats:2.8.0-alpine"
```

We're using SHA value for these images, as the upstream is mutable, while our copy is not, and the sync would fail when the upstream image is updated
```suggestion
- source: "nats@sha256:911802abf8c7fe0e6a087b1dd6cda8e2672cb54a0f378a8a9ec392154c2874cb"
  tag: "2.8.0-alpine"
```

---

Updated it to:
```
- source: "nats@sha256:3dfa6b8def0d3c784ef7827f0c21b17afd01be56773c68d5950e9f29172ae796"
  tag: "nats:2.8.2-alpine"
```